services/forest/src/util/restAPI.js

```diff
@@ -0,0 +1,26 @@
+const fetch = require('node-fetch');
+const config = require('config');
+
+const backendURL = config.services.backend;
+
+const makeGetRequest = endpoint => (
+  fetch(`${backendURL}/${endpoint}`)
+    .then(response => (
+      response.status === 200 ? response.json() : {
```

Comment:
Fix style for this [ternary](https://github.com/airbnb/javascript/issues/1281#issuecomment-276191835) or change it to an if statement

Reply:
Done